assets/pages/digital-subscription-checkout/digitalSubscriptionCheckoutReducer.js

```diff
@@ -202,6 +239,12 @@ function initReducer(initialCountry: IsoCountry) {
     email: user.email || '',
     firstName: user.firstName || '',
     lastName: user.lastName || '',
+    addressLine1: '',
+    addressLine2: '',
```

Comment:
do we need this to be nullable if it's optional?

Reply:
It's handled the same way as telephone below which is also optional in the form.